src/ui/common/src/components/integrations/numWorkflowsUsingMsg.tsx

```diff
@@ -0,0 +1,10 @@
+// Takes the number of workflows using the integration and returns a consistent string message.
+export function getNumWorkflowsUsingMessage(numWorkflowsUsing: number): string {
```

Comment:
We can probably make a function which returns the singular or plural version given a number and base word. i.e. 2 and resource would return "resources"